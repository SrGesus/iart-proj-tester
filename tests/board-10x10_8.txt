FD	FE	BB	FC	FC	FC	LV	VC	FD	FC
LH	FD	BC	FC	VD	LH	VC	LH	LV	LV
VE	VE	LV	BB	LH	BB	BE	BD	BC	LV
FC	VD	BD	VC	FD	VD	FB	FB	LV	LH
LH	VD	FC	FC	VE	VB	BE	VB	BB	BB
BE	BB	VE	BB	BE	FD	VE	VC	FD	LH
LV	BB	VB	LH	VD	LV	LV	BE	BE	BE
LV	BE	FB	BE	FE	FE	VB	VE	VC	VE
LV	FE	FB	LV	FB	VB	BE	VB	BB	VB
VE	LV	VC	FB	VD	LH	BB	FC	FD	VB
