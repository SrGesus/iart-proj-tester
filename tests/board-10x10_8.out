FB	FD	BB	FE	FB	FD	LH	VE	FB	FB
LV	FD	BE	FB	VD	LH	VE	LV	LV	LV
VD	VE	LV	BD	LH	BB	BC	BC	BE	LV
FB	VD	BC	VC	FD	VC	FB	FB	LV	LV
LV	VB	FE	FD	VE	VB	BE	VD	BC	BE
BD	BE	VB	BB	BE	FC	VD	VE	FB	LV
LV	BD	VC	LV	VD	LH	LH	BC	BC	BE
LV	BD	FE	BD	FE	FB	VB	VE	VB	VC
LV	FC	FB	LV	FB	VD	BE	VD	BC	VE
VD	LH	VC	FC	VD	LH	BC	FE	FD	VC
